src/view3d.cpp

```diff
@@ -228,8 +228,11 @@ std::unique_ptr<QGLFramebufferObject> View3D::allocIncrementalFramebuffer(int w,
     // for samples==1, so work around it by forcing 0, if possible
     fboFmt.setSamples(fmt.samples() > 1 ? fmt.samples() : 0);
     //fboFmt.setTextureTarget();
-    return std::unique_ptr<QGLFramebufferObject>(
-        new QGLFramebufferObject(w, h, fboFmt));
+    std::unique_ptr<QGLFramebufferObject> fbo;
+    fbo.reset(new QGLFramebufferObject(w, h, fboFmt));
+    if (fbo.get() && fbo->attachment()==QGLFramebufferObject::NoAttachment)
```

Comment:
BTW, I don't know how `fbo.get()` could ever return null so this seems overly defensive.  The only way for the constructor to error out is to throw.  If it can't be allocated it'll be a `std::bad_alloc`, otherwise whatever Qt-defined exception.  (Does qt even use exceptions?  AFAIK not.)